src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -1376,4 +1376,81 @@ static public int endOfShard(int shardIndex, int shardSize) {
     public enum IntervalBreakpointType {
         START_BREAKPOINT, END_BREAKPOINT
     }
+
+    /**
+     * Creates a {@link SimpleInterval} on a coordinate from the provided sequence dictionary.
+     *
+     * @param dictionary reference dictionary.
+     * @param contig     the new location contig name.
+     * @param start      the new location start base index.
+     * @param stop       the new location stop base index.
+     *
+     * @return never {@code null}.
+     * @throws UserException if there was some problem when creating the location.
+     */
+    public static SimpleInterval createInterval(final SAMSequenceDictionary dictionary, final String contig, final int start, final int stop) {
+        Utils.nonNull(dictionary, "null dictionary");
+        Utils.nonNull(contig, "null contig");
+        if (!SimpleInterval.isValid(contig, start, stop)) {
+            throw new UserException(String.format("Invalid coordinate: %s:%s-%s", contig, start, stop));
+        }
+        final int contigIndex = dictionary.getSequenceIndex(contig);
+        if (contigIndex == -1) {
+            throw new UserException(String.format("Invalid coordinate: %s:%s-%s (contig not in dictionary)", contig, start, stop));
+        }
+        final SAMSequenceRecord record = dictionary.getSequence(contigIndex);
+        if (record.getSequenceLength() < stop) {
+            throw new UserException(String.format("Invalid coordinate: %s:%s-%s (stop after sequence length %s)", contig, start, stop, record.getSequenceLength()));
+        }
+        return new SimpleInterval(dictionary.getSequence(contigIndex).getSequenceName(), start, stop);
+    }
+
+    /**
+     * Creates a {@link SimpleInterval} on an entire contig from the provided sequence dictionary
+     *
+     * @param dictionary  reference dictionary.
+     * @param contigIndex the new location contig index.
+     * @return never {@code null}.
+     *
+     * @throws UserException if there was some problem when creating the location.
+     */
+    public static SimpleInterval createOverEntireContig(final SAMSequenceDictionary dictionary, final int contigIndex) {
+        Utils.nonNull(dictionary, "null dictionary");
```

Comment:
"null dictionary" -> "dictionary is null"

Reply:
Done.